build/org.eclipse.cdt.managedbuilder.core/META-INF/MANIFEST.MF

```diff
@@ -39,7 +40,8 @@ Require-Bundle: org.eclipse.cdt.core;bundle-version="[8.3.0,9.0.0)",
  org.eclipse.ui;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.core.variables;bundle-version="[3.1.100,4.0.0)",
  org.eclipse.cdt.make.core;visibility:=reexport,
- org.eclipse.core.filesystem;bundle-version="1.2.0"
+ org.eclipse.core.filesystem;bundle-version="1.2.0",
+ org.eclipse.jdt.annotation;bundle-version="[2.2.7,3.0.0)";resolution:=optional
```

Comment:
@jonahgraham do we want to use Null annotations in CDT or should this be dropped?

Reply:
I think things are working here.